styles.css

```diff
@@ -0,0 +1,6 @@
+h1 {
+    color: blue;
+}
+h2 {
+    color: green;
```

Comment:
Dit moet dus die h3 zijn ipv een h2. Voor deze klant is oranje de company kleur, dus zou die kiezen :-)